pytorch_lightning/accelerators/accelerator.py

```diff
@@ -338,12 +338,21 @@ def lightning_module_state_dict(self) -> Dict[str, Union[Any, Tensor]]:
         return self.training_type_plugin.lightning_module_state_dict()
 
     def barrier(self, name: Optional[str] = None) -> None:
+        """
+        .. deprecated:: v1.5
+            This method is deprecated in v1.5 and will be removed in v1.7.
+            Please call training_type_plugin.barrier directly
+        """
```

Comment:
@awaelchli @justusschock  should we do a 2-version cycle for this? or remove in v1.6? or remove directly given the API is marked as experimental?

Reply:
I think we may want to go through a deprecation cycle for this. It's not impossible user is call accelerator.barrier() now

Reply:
I propose we mark this for removal in v1.6. So it's not a breaking change now but we could remove it faster because this API is more experimental

Reply:
yeah make sense

Reply:
1.6 sounds reasonable